components/drafts/drafts_link/drafts_link.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Did you meant to create this file, again?

Reply:
Yes sorry, good catch. It looks like the deletion of the original `drafts_link` files didn't get tracked. I've removed those extra files now 👍